src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -213,6 +285,21 @@ def child_sort_key(item: TraceEvent) -> List[int]:
         return [0]
 
 
+def count_performance_issues(trace_id: str, params: Mapping[str, str]) -> int:
+    transaction_query = QueryBuilder(
+        Dataset.Transactions,
+        params,
+        query=f"trace:{trace_id}",
+        selected_columns=[],
+        limit=MAX_TRACE_SIZE,
+    )
+    transaction_query.columns.append(
+        Function("sum", [Function("length", [Column("group_ids")])], "total_groups")
+    )
```

Comment:
Just to explain this one, group_ids are currently arrays.

So selecting just group_ids you get a bunch of rows like
[some group id, some other id]
[more group id]
[]
[another group id wow]

so running length first will get you:
2
1
0
1

Finally run sum to get `5`